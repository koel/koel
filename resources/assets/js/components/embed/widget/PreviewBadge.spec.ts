import { describe, expect, it } from 'vitest'
import { createHarness } from '@/__tests__/TestHarness'
import Component from './PreviewBadge.vue'

describe('previewBadge.vue', () => {
  const h = createHarness()

  it('renders properly', () => {
    const { html } = h.render(Component)
    expect(html()).toMatchSnapshot()
  })
})
