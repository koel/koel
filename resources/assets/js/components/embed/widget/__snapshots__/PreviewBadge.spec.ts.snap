// Vitest Snapshot v1

exports[`previewBadge.vue > renders properly 1`] = `<span class="text-xs uppercase font-semibold bg-white/10 rounded px-[5px] py-[1px] mt-[5px] border border-px border-white/10"> Preview </span>`;
